freqtrade/freqai/RL/BaseReinforcementLearningModel.py

```diff
@@ -156,6 +157,8 @@ def set_train_and_eval_environments(self, data_dictionary: Dict[str, DataFrame],
                                           render=False, eval_freq=len(train_df),
                                           best_model_save_path=str(dk.data_path))
 
+        self.tensorboard_callback = TensorboardCallback()
```

Comment:
This can go inside the `BaseReinforcementLearner.__init__()` so that we don't need to replicate it throughout.

Reply:
Looks like the multi_proc variant is our constraint here - we can't store the callback object inside the multiproc env, since there are a bunch of envs duplicated across various processes. Instead we need to keep the callback stored in the model, as you did here.